pkg/client/client_test.go

```diff
@@ -79,9 +80,9 @@ func TestConfig(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			client, err := Config(test.kubeconfig, test.kubecontext, "velero", test.QPS, test.burst)
-			assert.NoError(t, err)
+			require.NoError(t, err)
 			assert.Equal(t, test.expectedHost, client.Host)
-			assert.Equal(t, test.QPS, client.QPS)
+			assert.InDelta(t, test.QPS, client.QPS, 0.01)
```

Comment:
Why do we need to allow a difference between the expected and actual result here?